01-algorithms/08-algorithms-activity.md

```diff
@@ -1,209 +1,17 @@
-<!-- * topics: [python, pandas] (Checkpoints only. optional the topics for analyzing points) -->
-
-##### !question
-
-Suppose you are given the following question:
-
-Given an array of integers `nums` sorted in ascending order, find the starting and ending position of a given `target` value as a tuple.
-
-If the target is not found in the array, return (-1,-1)
-
-**Which of the following algorithmic approaches would best suit this problem?**
-##### !end-question
-
-##### !options
-
-a| Divide & Conquer
-b| Greedy
-c| Dynamic Programming
-
-##### !end-options
-
-##### !answer
-
-a|
-
-##### !end-answer
-
-<!-- other optional sections -->
-<!-- !hint - !end-hint (markdown, hidden, students click to view) -->
-<!-- !rubric - !end-rubric (markdown, instructors can see while scoring a checkpoint) -->
-<!-- !explanation - !end-explanation (markdown, students can see after answering correctly) -->
-
-### !end-challenge
-
-<!-- ======================= END CHALLENGE ======================= -->
-## Solving the problem
-
-As a class, we will pick 1-2 problems to solve (as time allows). 
+### Class Livecode
 
-We will first attempt to solve the problems as a small group, then come back to talk through any sticking points and work through possible solutions as a class. 
+Follow along in the following Replit as we livecode a recursive solution with and without Dynamic Programming: [Newman-Conway Recursively](https://replit.com/@adadev/Newman-Conway-Recursive)
 
-[Section A: Gas Station](https://replit.com/@adadev/gas-station#main.py) <br>
-[Section B: Minimum Operations](https://replit.com/@adadev/minimum-operations#README.md) <br>
-[Shortest Common Supersequence](https://replit.com/@adadev/shortest-common-supersequence#main.py) <br>
-[Find the First and Last Position of an Element](https://replit.com/@adadev/first-and-last-section-b#divide_and_conquer/find_first_and_last.py)
 
 ### Small Group Work
 
-Spend twenty minutes with your breakout room implementing a solution to the chosen problem using pseudocode or Python. 
-
-One person in the group should fork the replit and share a link with the rest of the group.
-
-If your entire group is feeling stuck, start by brainstorming some sample inputs and outputs. Try and think of at least one nominal case and two edge cases. Discuss any clarifying questions you have, and post them in your CS Fun section channel on Slack for instructor clarification.
-
-If your entire group is still feeling lost, discuss what is making you feel stuck and ask an instructor to come to your room for help! 
-
-### Class Livecode
-
-After small group work, we will come back together as a class to work through a solution together. Groups may ask questions, share where they got stuck, and/or share out their solutions.
+Fork the following Replit to work on in small groups: [Newman-Conway Iteratively with DP](https://replit.com/@adadev/Newman-Conway-Iterative)
```

Comment:
Perhaps in the replit in addition to specifying that it should be iterative, also include that it needs to be a dynamic programming approach

Reply:
Added, thank you!